tests/gdx-tests-android/AndroidManifest.xml

```diff
@@ -4,7 +4,7 @@
       android:versionCode="1"
       android:versionName="1.0"
       android:installLocation="preferExternal">
-	<uses-sdk android:minSdkVersion="8" android:targetSdkVersion="19"/>
+	<uses-sdk android:minSdkVersion="9" android:targetSdkVersion="19"/>
```

Comment:
The tests should probably be able to run on the same platform set as libGDX itself. I went ahead with the overrides instead of upping the minSdk. I have done some quick testing with my old Archos 48 (which is on Android 2.2) as well as in the emulator and most test seems to run fine.

That being said, Google has deprecated SDK 8 (and the associated images) so I'm wondering if we should keep supporting it moving forward.